sdk/cosmos/azure-cosmos-spark_3-0_2-12/docs/quick-start.md

```diff
@@ -0,0 +1,129 @@
+## Quick Start Guide for Cosmos DB Spark Connector Preview
+This tutorial is a quick start guide to show how to use Cosmos DB Spark Connector Preview to read from or write to Cosmos DB. Cosmos DB Spark Connector is based on Spark 3.0.x. 
+
+Throughput this quick tutorial we rely on Azure DataBricks Spark 3.0.1 and JupyterNotebook to show how to use the Cosmos DB Spark Connector.
+
+You can use any other Spark 3.0.x spark offering as well, also you should be able to use any language supported by Spark (PySpark, Scala, Java, etc), or any Spark interface you are familiar with (Jupyter Notebook, Livy, etc).
+
+### Prerequisites
+
+- An active Azure account. If you don't have one, you can sign up for a 
+  [free account](https://azure.microsoft.com/en-us/try/cosmosdb/).
+  Alternatively, you can use the
+  [use Azure Cosmos DB Emulator](local-emulator.md) for development and testing.
+- [Azure Databricks](https://azure.microsoft.com/en-us/services/databricks/) or any other Spark 3.0.x spark offering
+- (Optional) [SLF4J binding](https://www.slf4j.org/manual.html) is used to associate a
+  specific logging framework with SLF4J.
+
+SLF4J is only needed if you plan to use logging, please also download an SLF4J binding which will
+link the SLF4J API with the logging implementation of your choice. See
+the [SLF4J user manual](https://www.slf4j.org/manual.html) for more information.
+
+You need a Spark 3.0.1 Environment.
+
+Install Cosmos DB Spark Connector, in your spark Cluster `azure-cosmos-spark_3-0_2-12-4.0.0-beta.1.jar`
```

Comment:
Should we also add an aka.ms link for the "latest known good" version?

Reply:
we will add later once we have a landing page on the Microsoft docs. 
We have another work item for the ms side of docs. 
https://github.com/Azure/azure-sdk-for-java/issues/19573

I am hoping we get anfeldma-ms help for that.